nbsite/gallery/gen.py

```diff
@@ -0,0 +1,401 @@
+import os
+import glob
+import logging
+
+import requests
+import sphinx.util
+
+from .thumbnailer import notebook_thumbnail, execute
+
+# Try Python 3 first, otherwise load from Python 2
```

Comment:
Do we need nbsite to support python 2?

Reply:
I'd be very happy to say no.

Reply:
I think we can start clearing up this kind of logic when we see it.